### ISSUE TITLE
[FEAT] loss function 추가

### ISSUE THREAD

**Issue by yunjinchoidev** [dive-into-ai/lachesis#6]
## Background
- 기본 베이스라인에  loss function 몇 가지 추가
-

## To Do
- [x] focal_loss
- [x] label_smoothing
- [x] cross_entopy
- [x] mse_loss
- [x] bce_loss
- [x] bce_with_logits_loss
- [x] l1_loss
- [x] l2_loss
- [x] smooth_l1_loss

## See Also
-
